src/main/java/hw9/ValidateUtil.java

```diff
@@ -0,0 +1,39 @@
+package hw9;
+
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ValidateUtil {
+
+    private final static Logger log = Logger.getLogger(ValidateUtil.class.getName());
```

Comment:
Naming is not right. But as you don't use it, please remove it.
After removing don't forget to remove unused imports.